code/modules/mob/living/basic/space_fauna/spider/giant_spider/giant_spider_variants.dm

```diff
@@ -55,6 +121,62 @@
 		complete_text = "%SOURCE% wraps the wounds of %TARGET%.",\
 	)
 
+	AddElement(/datum/element/web_walker, /datum/movespeed_modifier/nurse_web)
+
+/**
+ * ### Tangle Spider
+ *
+ * A subtype of the giant spider which specializes in support skills.
+ * Nurses can place down webbing in a quarter of the time that other species can and can wrap other spiders' wounds, healing them.
+ * Note that it cannot heal itself.
+ */
+/mob/living/basic/giant_spider/tangle
+	name = "tangle spider"
+	desc = "Furry and brown, it makes you shudder to look at it. This one has dim brown eyes."
+	icon_state = "tangle"
+	icon_living = "tangle"
+	icon_dead = "tangle_dead"
+	gender = FEMALE
+	butcher_results = list(/obj/item/food/meat/slab/spider = 2, /obj/item/food/spiderleg = 8, /obj/item/food/spidereggs = 4)
+	maxHealth = 40
+	health = 40
+	melee_damage_lower = 1
+	melee_damage_upper = 1
+	poison_per_bite = 5
+	poison_type = /datum/reagent/toxin/acid
+	obj_damage = 40
+	web_speed = 0.25
+	speed = 4
+	player_speed_modifier = -3.1
+	web_type = /datum/action/cooldown/lay_web/sealer
+	menu_description = "Support spider variant specializing in contruction to protect their brethren, but has very low amount of health and deals low damage."
+
+/mob/living/basic/giant_spider/tangle/Initialize(mapload)
+	. = ..()
+	var/datum/action/cooldown/lay_web/solid_web/web_solid = new(src)
+	web_solid.Grant(src)
+
+	var/datum/action/cooldown/lay_web/web_passage/passage_web = new(src)
+	passage_web.Grant(src)
+
+	var/datum/action/cooldown/lay_web/web_spikes/spikes_web = new(src)
+	spikes_web.Grant(src)
+
+	var/datum/action/cooldown/lay_web/sticky_web/web_sticky = new(src)
+	web_sticky.Grant(src)
```

Comment:
All four of these abilities are always granted at the same time, I am wonder if there is some way of consolidating them into fewer buttons which does not compromise on UX... there might not be though.

Reply:
> All four of these abilities are always granted at the same time, I am wonder if there is some way of consolidating them into fewer buttons which does not compromise on UX... there might not be though. 

I think modsuits have this I would like to implemented in the future when more spiders come but for now I'm not to sure how I would do that and it isn't a huge issue right now